shelf/logger_creator.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import sys
+
+
+class LoggerCreator(object):
```

Comment:
I am guessing this is just out of scope for this but it looks like this could be updated to use this `shelf/background_utils.py`. Although not required.